pkg/cmd/server/start/admission_sync_test.go

```diff
@@ -16,6 +16,7 @@ import (
 var admissionPluginsNotUsedByKube = sets.NewString(
 	"AlwaysAdmit",            // from kube, no need for this by default
 	"AlwaysDeny",             // from kube, no need for this by default
+	"AlwaysPullImages",       // from kube, not enabled by default
```

Comment:
Remove the entry further down in the list.  Why don't we enable it by default?  I didn't because I didn't want to deal with behavior fallout.